CHANGES.rst

```diff
@@ -7,10 +7,67 @@ Version 5.0.0
 
 Released: XXXX-XX-XX
 
```

Comment:
We probably need a summary, especially since this is a major release. Users are going to want to know what is so major about it. I believe the dropping of Python 2 and removals of several packages should be mentioned.